examples/apps/presence-tracker/src/app.ts

```diff
@@ -2,27 +2,12 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
-
-import { Signaler } from "@fluid-experimental/data-objects";
-import { IFluidContainer, ContainerSchema } from "fluid-framework";
-import {
-	TinyliciousClient,
-	TinyliciousContainerServices,
-} from "@fluidframework/tinylicious-client";
+import { StaticCodeLoader, TinyliciousModelLoader } from "@fluid-example/example-utils";
 import { FocusTracker } from "./FocusTracker";
 import { MouseTracker } from "./MouseTracker";
+import { ITrackerAppModel, TrackerContainerRuntimeFactory } from "./containerCode";
 
-// Define the schema of our Container.
-// This includes the DataObjects we support and any initial DataObjects we want created
-// when the container is first created.
-const containerSchema: ContainerSchema = {
-	initialObjects: {
-		/* [id]: DataObject */
-		signaler: Signaler,
-	},
-};
-
-function renderFocusPresence(focusTracker: FocusTracker, div: HTMLDivElement) {
+export function renderFocusPresence(focusTracker: FocusTracker, div: HTMLDivElement) {
```

Comment:
Totally optional, but in some of our demos we split the render functions out to a separate view.ts file which can be kind of nice.